### PR TITLE
Resolved the issues with Travis

### PR DESCRIPTION
**Pull request by laconc**
Travis doesn't fail anymore!

* Now tests against Python 2.7 and 3.5
* Resolved a bunch of PEP8 issues
* Increased the max line length to 120

### CODE REVIEW

File: pycam/Importers/STLImporter.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def is_binary_input_format(source):
         return True
 
 
-def ImportModel(filename, use_kdtree=True, callback=None, **kwargs):
+def ImportModel(filename, use_kdtree=True, callback=None, numfacets=None, **kwargs):
```
Comment (sumpfralle) [inline pycam/Importers/STLImporter.py]: This missing variable made me aware of a problem, I forgot there. I fixed it now. Thus this change is not necessary anymore.
Thank you!

File: setup.cfg [hunk 2]
```diff
@@ -16,7 +16,7 @@ doc_files = Changelog
     examples/
 
 [flake8]
-max-line-length = 99
+max-line-length = 120
```
Comment (sumpfralle) [inline setup.cfg]: I am a friend of PEP-0008, which advocates a [limit of 99 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). Thus I would prefer to keep this value as it is, if you do not see good reasons for a change.

File: pycam/Importers/STLImporter.py (hunk 1 above)
Comment (laconc) [inline pycam/Importers/STLImporter.py]: Fantastic!

File: setup.cfg (hunk 2 above)
Comment (laconc) [inline setup.cfg]: I knew this would be controversial 🙂  99 it is!

**Comment by sumpfralle** [comment]
Cleanliness is back - thank you!

**Pull request merged**